test/fixtures/combined_vancouver_queens.json

```diff
@@ -949640,6 +949348,48 @@
       "bounding_box": "{\"min_lat\":49.2103949,\"max_lat\":49.2106953,\"min_lon\":-123.0838782,\"max_lon\":-123.0828887}"
     }
   },
+  {
+    "_index": "pelias",
+    "_id": "openstreetmap:venue:way/84971606",
+    "_type": "_doc",
+    "data": {
+      "name": {
+        "default": "Platforms 1 & 2"
```

Comment:
this is being brought in from `railway~platform+name`, maybe we should remove that since platform names will always be weird things like numbers and letters which don't make much sense out of context.

Reply:
added a second commit to remove these